build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'java'
```

Comment:
this is changed in order the plugin would also be able to compile scala sources (scala plugin can do both: Java and Scala, whereas 'java' seems to work only for Java)